### PR TITLE
feat: docker multi arch (ARM64 Support)

### PR DESCRIPTION
**Pull request by j0nl1**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -48,4 +48,3 @@ jobs:
           platforms: linux/amd64,linux/arm/v7
```
Comment (faddat) [inline .github/workflows/docker.yml:48]: can you please make this:

          platforms: linux/amd64,linux/arm64

?

armv7 is 32 bit

Comment (faddat) [reply .github/workflows/docker.yml:48]: https://docs.tendermint.com/master/nodes/running-in-production.html#validator-signing-on-32-bit-architectures-or-arm

**Comment by faddat** [comment]
Hi, I would be very happy to approve this because I've never liked working with ARG and ENV combinations and docker. 

I think that the only additional change needed here is to change linux/armv7 (think: really old raspberry pi) to either linux/arm64 (my preference, clarity) or linux/armv8.

Thank you!

**Comment by sascha1337** [comment]
as this is some topic atm, @faddat do you know why some chains use this mimalloc ?

https://github.com/terra-money/core/blob/ea682c41e7e71ba0b182c9e7f989855fb9595885/Dockerfile#L22-L35

File: .github/workflows/docker.yml (hunk 1 above)
Comment (j0nl1) [reply .github/workflows/docker.yml]: Thanks @faddat it was completely a typo. It updated now.

**Comment by faddat** [comment]
https://github.com/microsoft/mimalloc

**Review by faddat — APPROVED** [review]
This looks great.

**Comment by faddat** [comment]
@sascha1337 hey there sir!

I'm going to make an issue for mimalloc because I think that this Pr is good without it

**Comment by sascha1337** [comment]
@faddat ofc, just been curious and did fit in the context :D

**Review by JakeHartnell — APPROVED** [review]

**Comment by faddat** [comment]
I am just checking out the build now, and then I'll merge this, and **thank you!**

I love to see ergonomics improve.

@j0nl1 I bet that Osmosis could use these changes as well, if you're interested in making a PR.

**Pull request merged**